tst/regression/test_suites/advection_performance/advection_performance.py

```diff
@@ -78,8 +78,13 @@ def Analyse(self, parameters):
 
         perfs = np.array(perfs)
 
-        # Plot results
+        # Save performance metrics to file
+        with open('performance_metrics.txt', 'w') as writer:
+          writer.write("zone-cycles/s | Normalize Overhead | Meshblock Size\n")
+          for ind, mb_size in enumerate(mb_sizes):
+              writer.write("%14e %14e %14e\n" % (perfs[ind],perfs[0]/perfs[ind],mb_size))
```

Comment:
```suggestion
              writer.write("%.2e %.2e %d\n" % (perfs[ind],perfs[0]/perfs[ind],mb_size))
```
I'm not sure `%14e` has any effect. With `%.2e` we'll get three significant digits total, which should be fine in terms of accuracy for these kind of tests.

Reply:
Are these values standard metrics to print across multiple tests? If so, I think we should standardize the format. If not, @pgrete's suggestions aren't unreasonable as this would allow us to tell % level changes in performance which is sufficient for our purposes.

Reply:
Sounds good, and yes they are.